src/main/webapp/pipe.js

```diff
@@ -351,7 +351,7 @@ function generatePromotionsInfo(data, task) {
     if (data.showPromotions && task.status.promoted && task.status.promotions && task.status.promotions.length > 0) {
         var html = ["<div class='infoPanelOuter'>"];
         Q.each(task.status.promotions, function(i, promo) {
-            html.push("<div class='infoPanel'><div class='infoPanelInner'>");
+            html.push("<div class='infoPanel'><div class='infoPanelInner'><div class='promo-layer'>");
```

Comment:
Actually there is no wrapper div for showing promotions. It uses general
info panel layers. To specify any styles specifically to promotions info
for this I have included promo-layer.
On 11 May 2016 11:59 pm, "Tommy Tynjä" <notifications@github.com> wrote:

> In src/main/webapp/pipe.js
> <https://github.com/Diabol/delivery-pipeline-plugin/pull/178#discussion_r62901068>
> :
>
> > @@ -351,7 +351,7 @@ function generatePromotionsInfo(data, task) {
> >      if (data.showPromotions && task.status.promoted && task.status.promotions && task.status.promotions.length > 0) {
> >          var html = ["<div class='infoPanelOuter'>"];
> >          Q.each(task.status.promotions, function(i, promo) {
> > -            html.push("<div class='infoPanel'><div class='infoPanelInner'>");
> > +            html.push("<div class='infoPanel'><div class='infoPanelInner'><div class='promo-layer'>");
>
> What does the div class='promo-layer' add?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Diabol/delivery-pipeline-plugin/pull/178/files/e3fd206e4c5948deb3913ba33ebf60b47f5a1ad6#r62901068>
>